kedro/framework/session/session.py

```diff
@@ -327,7 +339,14 @@ def run(  # pylint: disable=too-many-arguments,too-many-locals
         # Report project name
         self._logger.info("** Kedro project %s", self._project_path.name)
 
-        save_version = run_id = self.store["session_id"]
+        if self._run_called:
+            raise KedroSessionError(
+                "A run has already been executed as part of the"
```

Comment:
```suggestion
                "A run has already been completed as part of the"
```
Maybe, to ease with the confusion mentioned in the previous comment.